web-common/src/features/file-explorer/FileExplorer.svelte

```diff
@@ -103,7 +103,7 @@
   ) {
     const isCurrentFile =
       removeLeadingSlash(fromDragData.filePath) ===
-      removeLeadingSlash($page.params.file);
+      removeLeadingSlash($page.params.file ?? "");
```

Comment:
Semantically, this doesn't make much sense. Better to use a guard clause to catch the `params.file === ""` case earlier.